modules/components/package.json

```diff
@@ -3,19 +3,23 @@
   "version": "3.0.0-beta.3",
   "description": "Data Portal Components",
   "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "publishConfig": {
     "access": "public"
   },
   "scripts": {
+    "build": "babel src --out-dir dist --copy-files --extensions \".tsx,.ts,.js\"",
     "build-storybook": "build-storybook -s ./public",
+    "cleanDist": "rm -rf dist",
     "dashboard": "STORYBOOK_DASHBOARD=true start-storybook -p 6060 -c .storybook -s ./public",
-    "flow": "flow",
+    "declarations": "ttsc --emitDeclarationOnly",
     "lint": "eslint src/**/*.js",
     "portal": "STORYBOOK_PORTAL=true start-storybook -p 7070 -c .storybook -s ./public",
-    "prepare": "rm -rf dist && babel src --out-dir dist --copy-files --extensions \".tsx,.ts,.js\"",
+    "postpack": "npm run declarations",
+    "prepare": "npm run cleanDist && npm run build",
```

Comment:
separated the type definitions to a packaging script, so the errors currently coming up from Flow code won't crash the bootstraping of modules.